modules/tensor_mechanics/test/tests/ics/volume_weighted_weibull/volume_weighted_weibull.i

```diff
@@ -0,0 +1,58 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 100
+  ny = 100
+[]
+
+[Problem]
+  solve = false
+[]
+
+[AuxVariables]
+  [u_vww]
+    order = CONSTANT
+    family = MONOMIAL
+  []
+[]
+
+[ICs]
+  [u_vww]
+    type = VolumeWeightedWeibull
+    legacy_generator = false
```

Comment:
we should probably remove this. The default is false and this option will be deprecated.

Reply:
Removed it.